core/templates/dev/head/css/oppia.css

```diff
@@ -77,6 +81,10 @@ button.md-button.md-default-theme.oppia-learner-continue-button:hover {
 .material-icons.md-dark {
   color: rgba(0, 0, 0, 0.54);
 }
+.middle{
```

Comment:
Hm. I think that's fine, actually. Can we leave it like that for now, and get rid of the ".middle" class? There's value in not having too many special cases.